index.html

```diff
@@ -17,21 +17,26 @@
             <li class="navbar-toggle">
               <img src="img/Union.png" alt="menubar" />
             </li>
+            <li>Portoflio</li>
+            <li>About</li>
+            <li>Contact</li>
```

Comment:
- [ ] Kindly fix the navigation in the mobile version
<img width="495" alt="Screenshot 2023-02-25 at 03 29 19" src="https://user-images.githubusercontent.com/81444338/221354611-70ca66c4-1f1d-44f5-afe4-fad920c0472b.png">

- [ ] On the desktop give a margin it is too close

Your Design
<img width="1440" alt="Screenshot 2023-02-25 at 03 29 00" src="https://user-images.githubusercontent.com/81444338/221354825-623db0b9-2367-4666-8575-3d95cff49afd.png">

Figma design
<img width="1440" alt="Screenshot 2023-02-25 at 03 43 56" src="https://user-images.githubusercontent.com/81444338/221357149-e4bdc527-4aab-4878-89d8-9995f5bbad04.png">

Reply:
Done!